src/app/cube/details/details.component.ts

```diff
@@ -109,6 +110,14 @@ export class DetailsComponent implements OnInit, OnDestroy {
           return file;
         }
       );
+      this.children = this.learningObject.children.filter(
```

Comment:
Can we add a FIXME to this, stating that it should be removed once the service logic is updated? It's just a temporary addition